### ISSUE TITLE
Build failure with gcc 4.8

### ISSUE THREAD

**Issue by ffontaine** [intel/tinycbor#214]
We have the following build failure on builroot with gcc 4.8 since bump to version 0.6.0 and https://github.com/intel/tinycbor/commit/e2a4ed135c4d9101c4df83f2dd033cd249b6ef07:
```
/home/buildroot/autobuild/instance-1/output-1/host/bin/arm-none-linux-gnueabi-gcc -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -I./src -std=gnu99 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64  -Os -g0 -D_FORTIFY_SOURCE=1 -Werror=incompatible-pointer-types -Werror=implicit-function-declaration -Werror=int-conversion -fPIC  -c -o src/cborerrorstrings.pic.o src/cborerrorstrings.c
cc1: error: -Werror=incompatible-pointer-types: no option -Wincompatible-pointer-types
cc1: error: -Werror=int-conversion: no option -Wint-conversion
```

Should I add a dependency on gcc >= 5 or would it be possible/acceptable to enable those flags only in "development" mode?

Full build log: http://autobuild.buildroot.org/results/a9f/a9fe64c42bb96f9e7b4af3050464f6570c1c00fa/build-end.log

**Comment by thiagomacieira** [comment]
GCC 4.8 is now over 6.5 years old. Is it important to you?

Either solution is fine for me, so feel free to apply what's easiest for you.

**Comment by ffontaine** [comment]
Thanks for your feedback, I sent #215.

**Comment by thiagomacieira** [comment]
Fixed in #215.
